packages/sdks/src/functions/is-from-trusted-host.ts

```diff
@@ -0,0 +1,23 @@
+const DEFAULT_TRUSTED_HOSTS = [
+  '*.beta.builder.io',
+  'beta.builder.io',
+  'builder.io',
+  'localhost',
+  'qa.builder.io',
+];
+
+export function isFromTrustedHost(
```

Comment:
Could we also get the unit tests you added to the gen1 SDK for this function? 🙏🏽